lib/services/ngsi/entities-NGSI-v2.js

```diff
@@ -336,6 +339,16 @@ function removeHiddenAttrsFromMultiEntity(entities, typeInformation) {
                 explicitAttrsList.push(attr.name);
             });
         }
+    } else if (typeInformation.explicitAttrs && typeof typeInformation.explicitAttrs === 'string') {
+        entities.forEach((entity) => {
+            const attsArray = pluginUtils.extractAttributesArrayFromNgsi2Entity(entity);
+            const ctx = jexlParser.extractContext(attsArray);
+            const res = jexlParser.applyExpression(typeInformation.explicitAttrs, ctx, typeInformation);
+            explicitAttrsList = explicitAttrsList.concat(res);
```

Comment:
why concat with []? ... res is the propper array, isn't it? it is due to some type checking, or to ensure it is an array?

Reply:
In this case (multientity) there is a loop and several res arrays should be accumulated into explicitAttrsList